sync.js

```diff
@@ -552,6 +604,13 @@ function WifiPeer (connection, info) {
   return info
 }
 
+function WebsocketPeer (connection, info) {
+  info.type = 'ws'
```

Comment:
Could we maybe call this type `websocket` just to avoid potential confusion with the protocol (`ws` or `wss`).